components/engine-typescript/src/main/java/org/eclipse/dirigible/TypeScriptEndpoint.java

```diff
@@ -9,9 +9,10 @@
  * SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and Eclipse Dirigible contributors
  * SPDX-License-Identifier: EPL-2.0
  */
-package org.eclipse.dirigible.components.engine.javascript.endpoint;
+package org.eclipse.dirigible;
```

Comment:
is there a reason for shortening the package instead of e.g. org.eclipse.dirigible.components.engine.typescript

Reply:
No, I noticed it as well and will change it